osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -143,7 +132,7 @@ public virtual void Seek(double time)
         public void Reset(bool startClock = false)
         {
             // Manually stop the source in order to not affect the IsPaused state.
-            AdjustableSource.Stop();
+            decoupledClock.Stop();
```

Comment:
I don't generally have any strong opinions on the clock renames - I don't feel like there's really any difference in readability between the old and new names - but I will say that it makes some inline comments like this one kind of confusing to read (because `SourceClock` also exists in this class, this stopping `decoupledClock` next to a comment mentioning "stopping the source" looks like a bug almost).

Class probably needs a pass on inline comments to update textual references. A revert back to the old names could also work, I guess?

Reply:
Hmmmmm, decoupled clocks are always intended to be wrappers over a source, so it still makes sense to me.

Reply:
Going to leave this for now as I think it will make more sense further down the line.